processing/src/main/java/io/druid/segment/ColumnSelectorFactory.java

```diff
@@ -30,4 +31,5 @@
   public FloatColumnSelector makeFloatColumnSelector(String columnName);
   public LongColumnSelector makeLongColumnSelector(String columnName);
   public ObjectColumnSelector makeObjectColumnSelector(String columnName);
+  public ValueType columnType(String columnName);
```

Comment:
is it also possible ot add some javadocs? I know the current code doesn't have any but it is nice to have good javadocs on interfaces. For this particular method, I think we should document when nulls are expected, and when actual values are expected